manifests/fluentd/fluentd-mixed-cfgs-output-es.yaml

```diff
@@ -54,7 +54,7 @@ metadata:
 spec: 
   outputs: 
   - elasticsearch:
-      host: elasticsearch-logging-data.kubesphere-logging-system.svc
+      host: elasticsearch-logging-data.fluent.svc
```

Comment:
we can keep the es namespace as it is before : elasticsearch-logging-data.kubesphere-logging-system.svc